projects/super_res/super_res.py

```diff
@@ -0,0 +1,203 @@
+import xarray as xr
+import numpy as np
+import fv3viz
+from vcm.catalog import catalog
+import matplotlib.pyplot as plt
+import os
+import cartopy.crs as ccrs
+
+GRID = catalog["grid/c384"].read()
+
+DPI = 800
```

Comment:
nit: I would prefer having this as an argument to the function, perhaps with this as a default value.